apps/admin-x-activitypub/src/components/Profile.tsx

```
@@ -326,45 +340,50 @@
                             </span>
-                            {(userProfile?.summary || attachments.length > 0) && (
+                            {(account?.bio || customFields.length > 0) && (
                                 <div ref={contentRef} className={`ap-profile-content transition-max-height relative text-[1.5rem] duration-300 ease-in-out [&>p]:mb-3 ${isExpanded ? 'max-h-none pb-7' : 'max-h-[160px] overflow-hidden'} relative`}>
                                     <div
-                                        dangerouslySetInnerHTML={{__html: userProfile?.summary ?? ''}}
+                                        dangerouslySetInnerHTML={{__html: account?.bio ?? ''}}
                                         className='ap-profile-content mt-3 text-[1.5rem] [&>p]:mb-3'
                                     />
-                                    {attachments.map(attachment => (
+                                    {customFields.map(customField => (
                                         <span className='mt-3 line-clamp-1 flex flex-col text-[1.5rem]'>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)